example/app/app.component.html

```diff
@@ -33,5 +40,31 @@ <h1>&lt;angular-custom-modal&gt;</h1>
         </pre>
       </ng-template>
     </modal>
+
+    <modal #htmlInsideModalWhitNestedModal>
+      <ng-template #modalHeader>
+        <button class="close" data-dismiss="modal" type="button" aria-label="Close"
+                (click)="htmlInsideModalWhitNestedModal.close()">×
+        </button>
+        <h2>HTML inside modal whit nested modal</h2>
+      </ng-template>
+      <ng-template #modalBody>
+        I'm HTML who's living on the &lt;app.component.html&gt;<br>
+        This is for testing that ESC key press only close one modal.<br>
+        <button class="big" (click)="nestedModal.open()"
+        style="background-color: #4aabc9;">Modal with nested modal</button>
```

Comment:
When it's no longer a WIP, all styles should be moved inside their respective SCSS files. Additionally, try using the default styles rather than creating your own ones. It helps distinguish the needed CSS from the redundant one.

Reply:
Just use a diferent color, 'cause de default one was the same a modal background. Can you help with this?

Reply:
solved